trade-finance-manager-api/src/graphql/reducers/mappings/deal/mapTotals.js

```diff
@@ -20,23 +30,28 @@ const mapTotals = (facilities) => {
     // - Cash and Contingent facility total is `value`
 
     return Number(facilitySnapshot.value);
-  });
+  }));
 
   const formattedFacilitiesValue = formattedNumber(facilitiesValue.reduce((a, b) => a + b));
 
   totals.facilitiesValueInGBP = `GBP ${formattedFacilitiesValue}`;
 
   // total ukef exposure for all facilities
   const ukefExposureArray = [
-    ...facilities.map((f) => {
+    ...await Promise.all(facilities.map(async (f) => {
```

Comment:
this is getting a little messy here. can we create a variable instead? and then add the variable here?

Reply:
done